src/data/requests.js

```diff
@@ -61,6 +61,7 @@ export async function create(request) {
 }
 
 export async function update(request) {
+  request.photo_id = request.photo_id
```

Comment:
just checking but it looks like in src/components/CreateOrUpdateRequest.svelte requests.photoID is being used, but I'm not sure if that is the same "type" since we now have rest and gql.

Reply:
Right, the create function hasn't been migrated yet.